src/app/api/v1/signUp/route.ts

```diff
@@ -1,16 +1,18 @@
 import bcrypt from "bcryptjs";
+import { NextRequest, NextResponse } from "next/server";
 import dbConnect from "../../../../lib/dbConnect";
+import { verifyOtp } from "../../../../helpers/verifyOtp";
 import UserModel from "../../../../model/User";
 
-export async function POST(req: Request) {
+export async function POST(req: NextRequest) {
   await dbConnect();
   try {
-    const { username, email, password } = await req.json();
+    const { username, email, password, otp } = await req.json();
```

Comment:
Add validation to check if the password meets specific strength criteria before hashing it.